lint/eslint.bzl

```diff
@@ -186,17 +195,19 @@ def _eslint_aspect_impl(target, ctx):
     files_to_lint = filter_srcs(ctx.rule)
 
     if ctx.attr._options[LintOptionsInfo].fix:
-        patch, report, exit_code, info = patch_and_report_files(_MNEMONIC, target, ctx)
+        outputs, info = patch_and_output_files(_MNEMONIC, target, ctx)
         if len(files_to_lint) == 0:
-            dummy_successful_lint_action(ctx, report, exit_code, patch)
+            dummy_successful_lint_action(ctx, outputs.human.stdout, outputs.human.exit_code, outputs.patch)
         else:
-            eslint_fix(ctx, ctx.executable, files_to_lint, patch, report, exit_code)
+            eslint_fix(ctx, ctx.executable, files_to_lint, outputs.patch, outputs.human.stdout, outputs.human.exit_code)
     else:
-        report, exit_code, info = report_files(_MNEMONIC, target, ctx)
+        outputs, info = output_files(_MNEMONIC, target, ctx)
         if len(files_to_lint) == 0:
-            dummy_successful_lint_action(ctx, report, exit_code)
+            dummy_successful_lint_action(ctx, outputs.human.stdout, outputs.human.exit_code)
         else:
-            eslint_action(ctx, ctx.executable, files_to_lint, report, exit_code)
+            eslint_action(ctx, ctx.executable, files_to_lint, outputs.human.stdout, outputs.human.exit_code)
+
+    eslint_action(ctx, ctx.executable, files_to_lint, outputs.machine.stdout, outputs.machine.exit_code, format = ctx.attr._formatter)
```

Comment:
same comment for eslint as for shellcheck, https://github.com/aspect-build/rules_lint/pull/331/files#r1680423092